app/src/main/java/org/mozilla/fenix/sync/SyncedTabsViewHolder.kt

```diff
@@ -32,6 +32,7 @@ sealed class SyncedTabsViewHolder(itemView: View) : RecyclerView.ViewHolder(item
     abstract fun <T : AdapterItem> bind(item: T, interactor: SyncedTabsView.Listener)
 
     class TabViewHolder(itemView: View) : SyncedTabsViewHolder(itemView) {
+        private val binding = SyncTabsListItemBinding.bind(itemView)
```

Comment:
are we keeping this...or..

Reply:
There are 2 more similar instances. I think it might be a rebase issue.

Reply:
Thanks. Removed those now.